### ISSUE TITLE
multi alertmanager-bot with single telegram token

### ISSUE THREAD

**Issue by vtolstov** [metalmatze/alertmanager-bot#16]
I have problem:)
i have 4 dc that contains 2 prometheus servers with 2 alertmanager with 2 alertmanager-bots
1 server contains: promethes+alertmanager+alertmanager-bot

but when i'm for example get /alerts command to bot, i'm receive only one answer from one dc.
I don't want to create many bots, how can i work with multi dc mode ?

**Comment by Maxpain177** [comment]
I have the same problem

**Comment by vtolstov** [comment]
i'm contact telegram team and the best way to create application tokens and use it for bots ( https://my.telegram.org/apps )

**Comment by Maxpain177** [comment]
@vtolstov can you please describe your approach?

**Comment by vtolstov** [comment]
when you use application token each app with token from telegram POV is the different client, so you can distinguish it and control

**Comment by timmy59100** [comment]
@vtolstov Could you please describe that some more? I don't get which token you generate how?

**Comment by vtolstov** [comment]
Try to check web page from telegram where you can get app keys

**Comment by timmy59100** [comment]
@vtolstov Do you mean this? I can generate only one app. And api_hash is the token for alermanager-bot?
![image](https://user-images.githubusercontent.com/7300669/102910324-aa5f4000-447a-11eb-80c4-42a6dd99000e.png)

**Comment by vtolstov** [comment]
Yes, try to connect with two apps with the same keys, it must looks like different clients
